tools/enso4igv/src/test/java/org/enso/tools/enso4igv/EnsoSbtProjectTest.java

```diff
@@ -36,6 +37,23 @@ public void testLanguageServerProject() throws Exception {
 
     var javaGroups = s.getSourceGroups("java");
     assertEquals("1 bench, 2 tests, 4 main: " + Arrays.toString(javaGroups), 7, javaGroups.length);
+
+    var javaFile = root.getFileObject("src/main/java/MainJava.java");
+    assertNotNull("Main java found", javaFile);
+    var javaCp = ClassPath.getClassPath(javaFile, ClassPath.SOURCE);
+    assertNotNull("java classpath found", javaCp);
+
+    assertNotNull("Main java is on source path", javaCp.findResource("MainJava.java"));
+    assertNotNull("Main scala is on source path", javaCp.findResource("MainScala.scala"));
+    assertNull("Test scala is not on source path", javaCp.findResource("TestScala.scala"));
+
+    var scalaTestFile = root.getFileObject("src/test/scala/TestScala.scala");
+    assertNotNull("Test scala found", scalaTestFile);
+    for (var g : javaGroups) {
+      if (g.contains(scalaTestFile)) {
+        assertEquals("test/scala", g.getName());
+      }
```

Comment:
If no group contains the file, it will still succeed but that does not seem correct, is it?

I think we should `assert` that it is found in at least one place. Unless `getFileObject` implies that? But I think it's still better to just make it explicit.

Reply:
The real problem was that the files from `test/` directory were also contained in `main/` groups - that's what has to be prevented.